index.html

```diff
@@ -45,7 +45,7 @@ <h2 class="subtitle bold"><q>Hello! Sharing world</q></h2>
                 <p class="description">
                     A joyful celebration in the value of openess and sharing,
                     creating a positive change with people from over 10 countries
-                    is taking place in July, in Benin.
+                    is taking place in July, in Cotonou, Benin.
                 </p>
                 <div class="hero-event">
```

Comment:
### I write to you the review for this part at these lines here as I can not see all the code of the page 😊

- [ ] Kindly 😃 it will be better if you remove the footer from the index page in the mobile version as you see in the original design the footer is not found after the button of more,so let's try to remove the footer from the index in the mobile version to make it nice like the Behance design 💯🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|  ![image](https://github.com/lily4178993/leaders-week/assets/83360636/8abe9864-d1f1-4dc0-af16-e3021fcc9530)  |   ![image](https://github.com/lily4178993/leaders-week/assets/83360636/cb2f9d93-f7e6-4d1e-96d5-0ab26b7f7221)|